strr-platform-web/app/components/form/platform/ContactInfo.vue

```diff
@@ -86,89 +86,59 @@ watch(secondaryRep, async () => {
       :ui-radio="{ inner: 'space-y-2' }"
     />
     <div v-if="isCompletingPartyRep !== undefined" class="space-y-10">
-      <div v-if="!isCompletingPartyRep" class="space-y-10">
-        <ConnectPageSection
-          class="bg-white"
-          :heading="{ label: tPlat('section.title.completingParty'), labelClass: 'font-bold md:ml-6' }"
-        >
-          <UForm
-            ref="compPartyRef"
-            :schema="getContactSchema(true)"
-            :state="completingParty"
-          >
-            <FormCommonContact
-              v-model:first-name="completingParty.firstName"
```

Comment:
Would this be a good candidate for defineModel()?

Reply:
It is using that, but since this component has multiple model variables you specify each one with the `v-model:..`

Are you suggesting `v-model=completingParty` instead? That may work too, I didn't try that